app/src/renderer/components/staking/UndelegationModal.vue

```diff
@@ -29,10 +29,13 @@
         type="number"
         placeholder="Amount"
       />
+      <p v-if="maximum > 0">
+        {{ denom }}s delegated: {{ atoms(maximum) }}
+      </p>
       <tm-form-msg
-        v-if="liquidAtoms === 0"
-        :msg="`doesn't have any ${denom}s`"
-        name="Wallet"
+        v-if="maximum === 0"
+        :msg="`don't have any delegation to this validator`"
```

Comment:
```suggestion
        :msg="`don't have anything delegated to this validator`"
```

Reply:
what about :msg="`don't have any ${bondDenom} delegated to this validator`"

Reply:
yeah that works too. the trouble is that `atom` should have an `s` and `stake` should not 😆

Reply:
#grammarisweird